app/controllers/api/transformation_mappings_controller.rb

```diff
@@ -0,0 +1,16 @@
+module Api
+  class TransformationMappingsController < BaseController
+    def create_resource(type, id, data = {})
+      raise BadRequestError, "Must specify mappings" unless data["mappings"]
+      data["mappings"] = data["mappings"].each_with_object({}) do |(k, v), new_mappings|
+        begin
+          klass = collection_config.klass(k)
```

Comment:
@jntullo does `collect_config` already map for example `cluster` to `EmsCluster`?
What about the GET - where do you do the translation?

Reply:
@bzwei Yeah, it maps to `EmsCluster`, as defined in the [api.yml](https://github.com/ManageIQ/manageiq-api/blob/master/config/api.yml#L561-L568) - I forgot about the reverse translation (on `GET`), but I will add that in